sw/device/silicon_creator/lib/sigverify/spx_verify.c

```diff
@@ -62,6 +62,19 @@ static const uint32_t kSpxVerifyShares[kSigverifySpxRootNumWords] = {
     0x9eabb3c3,
 };
 
+/**
+ * Domain-separation prefix for SPHINCS+ with no prehashing.
+ *
+ * The domain separation prefix is the following byte sequence:
+ *   0x00 || len(ctx) || ctx
+ *
+ * In our case, `ctx` is always the empty string, so the length is 0.
+ */
+static const uint8_t kSpxVerifyPureDomainSep[] = {
```

Comment:
This sgtm. The sequence of three arguments gets reduced to two due to `len(ctx) == 2`.  This matches the proposed specification.